server/store.go

```diff
@@ -107,6 +107,18 @@ func (s *storeInfo) getLabelValue(key string) string {
 	return ""
 }
 
+func (s *storeInfo) getLocationID(keys []string) string {
+	id := ""
+	for _, k := range keys {
+		v := s.getLabelValue(k)
+		if len(v) == 0 {
+			return ""
```

Comment:
So we should forbid the missing label in TiKV, and if TiKV reports the label without a value, we should log an error and don't add this label too.